tests/languages/shell-session/issue2685.test

```diff
@@ -0,0 +1,29 @@
+/home/user$ echo "Hello World"
+Hello World
+/home/user$ exit
+
+----------------------------------------------------
+
+[
+	["command", [
+		["info", [
+			["path", "/home/user"]
+		]],
+		["shell-symbol", "$"],
+		["bash", [
+			["builtin", "echo"],
+			["string", ["\"Hello World\""]]
+		]]
+	]],
+
+	["output", "Hello World\r\n"],
+	["command", [
+		["info", [
+			["path", "/home/user"]
+		]],
+		["shell-symbol", "$"],
+		["bash", [
+			["builtin", "exit"]
+		]]
+	]]
+]
```

Comment:
Nit: new line

Reply:
I used the `npm run test:languages -- --accept` command to insert the token stream. With or without final new line, I don't mind, but if it's something that bothers you, then I'll change the implementation.

Reply:
lol it doesn't really bother me but I've noticed you making that change elsewhere so I figured we wanted consistency.

Reply:
Ahh kk. I only make that change everywhere because I configured my IDE that way.

If we really want that consistency, then I'll just reformat all test files. Only takes a second but will probably change ~500 files. Maybe it's not that necessary.